specs/commandline/inspect.md

```diff
@@ -169,14 +169,14 @@ localhost:5000/net-monitor@sha256:ca5427b5567d3e06a72e52d7da7dabfac484efe37a5380
     │   ├── user defined attributes
     │   │   └── io.wabbit-networks.buildId: 123
     │   ├── unsigned attributes
-    │   │   ├── timestamp signature
-    |   │   │   ├── timestamp: [Fri Jun 23 22:04:31 2023, Fri Jun 23 22:04:31 2023]
-    |   │   │   └── certificates
-    |   │   │       └── SHA256 fingerprint: d2f6e46ded7422ccd1d440576841366f828ada559aae3316af4d1a9ad40c7828
-    |   │   │           ├── issued to: wabbit-com Software Timestamp
-    |   │   │           ├── issued by: wabbit-com Software Trusted Timestamping
-    |   │   │           └── expiry: Fri Oct 13 23:59:59 2034
-    │   │   └── io.cncf.notary.signingAgent: notation/1.0.0
+    │   │   ├── signing agent: notation/1.0.0                               //client version
+    │   │   └── timestamp signature
+    │   │       ├── timestamp: [Fri Jun 23 22:04:31 2023, Fri Jun 23 22:04:31 2023]
```

Comment:
I am wondering why this timestamp does not uses nanosecond but only timestamp under unsignedAttributes?

Reply:
`unsignedAttributes` is for JSON output. In `text` output, it is `unsigned attributes`. `timestamp` is under unsigned attributes, as this is a text result for humans, so nanoseconds may not be helpful. It follows a predefined format called `ANSIC`, which doesn't include nanoseconds.